pkg/pillar/types/zedroutertypes.go

```diff
@@ -870,6 +870,7 @@ type DhcpConfig struct {
 	DomainName string
 	NtpServer  net.IP
 	DnsServers []net.IP // If not set we use Gateway as DNS server
+	Type       NetworkType
```

Comment:
I wouldn't mind seeing the comment below added here as well; // IPv4 or IPv6 or Dual stack

Reply:
added